### PR TITLE
Rewrite Integration Branch

### PR DESCRIPTION
**Pull request by gundams-and-dragons**

### CODE REVIEW

File: package-old.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (mklucsarits) [inline package-old.json:1]: Maybe it was committed by accident but I don't think we need to keep the -old version of these files around. They'll be available in the commit history if we need to reference them again.

File: src/locales/de/main.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
 {
```
Comment (mklucsarits) [inline src/locales/de/main.json:1]: Do we need these files even though we have the `i18n.ts` file? Unless there's a process keeping the two i18n sources in sync, it doesn't seem wise to have them in more than one place.

File: src/players/DesktopArticleVideoPlayer/UnmuteButton.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	color: ${WDSVariables.wdsColorDarkBlueGray};
+	font-size: ${WDSVariables.wdsFontSizeXs};
+	font-weight: ${WDSVariables.wdsFontWeightBold};
+`;
+
+const UnmuteButton: React.FC = () => {
+	const { player } = useContext(PlayerContext);
+	const [muted, setMuted] = useState(true);
+	const unmute = () => {
+		console.log(player);
```
Comment (mklucsarits) [inline src/players/DesktopArticleVideoPlayer/UnmuteButton.tsx:32]: If this is for debugging, don't forget to remove it later on

File: src/players/MobileArticleVideoPlayer/OffScreenOverlay/OffScreenOverlay.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	height: 100%;
+	pointer-events: none;
+	z-index: 5;
+`;
+
+const OffScreenOverlay: React.FC = () => {
+	const playing = usePlaying();
+	const controlbar = document.querySelector<HTMLElement>('.jw-controlbar');
+
+	if (!playing) {
```
Comment (mklucsarits) [inline src/players/MobileArticleVideoPlayer/OffScreenOverlay/OffScreenOverlay.tsx:26]: Maybe easier to read:
```suggestion
	if (controlbar) {
		controlbar.style.visibility = playing ? 'visible' : 'hidden';
	}
```

File: src/players/MobileArticleVideoPlayer/OffScreenOverlay/OffScreenOverlay.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+		if (controlbar) controlbar.style.visibility = 'hidden';
+	} else {
+		if (controlbar) controlbar.style.visibility = 'visible';
+	}
```
Comment (mklucsarits) [inline src/players/MobileArticleVideoPlayer/OffScreenOverlay/OffScreenOverlay.tsx:30]: Maybe easier to read:
```suggestion
	if (controlbar) {
		controlbar.style.visibility = playing ? 'visible' : 'hidden';
	}
```

File: src/players/shared/JwPlayerWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+declare let window: WindowJWPlayer;
+
+/**
+ * gets the android player if user is on an android device browser
+ */
+const getDefaultPlayerUrl = () => {
+	return navigator.userAgent.match(/android/i)
+		? 'https://cdn.jwplayer.com/libraries/MFqndUHM.js'
+		: 'https://content.jwplatform.com/libraries/VXc5h4Tf.js';
```
Comment (mklucsarits) [inline src/players/shared/JwPlayerWrapper.tsx:18]: I think these URLs would be nice to have in a consts.ts file.

**Review by p-forfa — APPROVED** [review]

**Review by JoshuaRogan — APPROVED** [review]

**Pull request merged**